aviary/examples/run_reserve_mission_multiphase_time_and_range.py

```diff
@@ -0,0 +1,179 @@
+"""
+This is a slightly more complex Aviary example of running a coupled aircraft design-mission optimization.
+It runs the same mission as the `run_basic_aviary_example.py` script, but it uses the AviaryProblem class to set up the problem.
+This exposes more options and flexibility to the user and uses the "Level 2" API within Aviary.
+
+We define a `phase_info` object, which tells Aviary how to model the mission.
+Here we have climb, cruise, and descent phases.
+We then call the correct methods in order to set up and run an Aviary optimization problem.
+This performs a coupled design-mission optimization and outputs the results from Aviary into the `reports` folder.
+"""
+import aviary.api as av
+from example_phase_info import phase_info
+
```

Comment:
I recommend doing a deepcopy on the phase_info after importing it.

If you don't do this, the phase_info in memory in your process is modified, and will end up with the reserve missions in it. You can end up with a situation where a test that runs after the test and imports example_phase_info could fail because of the extra unexpected phases in it.